lib/Item/ValueUrlGenerator/TaxonValueUrlGenerator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Item\ValueUrlGenerator;
+
+use Netgen\Layouts\Item\ExtendedValueUrlGeneratorInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+/**
+ * @implements \Netgen\Layouts\Item\ExtendedValueUrlGeneratorInterface<\Sylius\Component\Taxonomy\Model\TaxonInterface>
+ */
+final class TaxonValueUrlGenerator implements ExtendedValueUrlGeneratorInterface
+{
+    public function __construct(private readonly UrlGeneratorInterface $urlGenerator) {}
```

Comment:
`private readonly` sneaked back in :)

Reply:
Fixed with b2ba9b2c244e83fb1444dbbee7d70499efebfd08